WooCommerce/WooCommerceTests/ViewRelated/Products/Edit Product/ProductFormActionsFactory+VisibilityTests.swift

```diff
@@ -160,6 +160,36 @@ final class ProductFormActionsFactory_VisibilityTests: XCTestCase {
         XCTAssertFalse(factory.settingsSectionActions().contains(.briefDescription))
         XCTAssertTrue(factory.bottomSheetActions().contains(.editBriefDescription))
     }
+
+    // MARK: - Downloadable Files
+
+    func testDownloadableFilesRowIsVisibleForDownloadableProductWithNonEmptyDownloadableFiles() {
```

Comment:
The same suggestion about using the underscores. 👍

Reply:
updated